### PR TITLE
Fix to load app's ini files after all other extensions are loaded

### LINKED ISSUE #538: php config file order
As php takes the last ini value set, in frankenphp/conf.d

The order currently is:

app.dev.ini
app.ini

Expected would be:

app.ini
app.dev.ini

Should app.dev.ini be renamed so it is loaded after app.ini?  Currently there is only one setting in app.dev.ini but more could be added and I believe the expected behavior would be it take precedence over settings in app.ini.

### PR DESCRIPTION
**Pull request by leroy0211**
Fixes #538 by specifying a different `conf.d` directory (`app.conf.d`) for loading the ini files. 

Renamed the files to numerical prefixes so they are loaded in the correct order. Default first, then environment specific. 

After running a dev build, this is the output of `php --ini`

```
Configuration File (php.ini) Path: /usr/local/etc/php
Loaded Configuration File:         /usr/local/etc/php/php.ini
Scan for additional .ini files in: /usr/local/etc/php/conf.d:/usr/local/etc/php/app.conf.d
Additional .ini files parsed:      /usr/local/etc/php/conf.d/docker-php-ext-apcu.ini,
/usr/local/etc/php/conf.d/docker-php-ext-intl.ini,
/usr/local/etc/php/conf.d/docker-php-ext-opcache.ini,
/usr/local/etc/php/conf.d/docker-php-ext-sodium.ini,
/usr/local/etc/php/conf.d/docker-php-ext-xdebug.ini,
/usr/local/etc/php/conf.d/docker-php-ext-zip.ini,
/usr/local/etc/php/app.conf.d/10-app.ini,
/usr/local/etc/php/app.conf.d/20-app.dev.ini
```

### CODE REVIEW

File: frankenphp/conf.d/10-app.ini
Comment (leroy0211) [inline frankenphp/conf.d/10-app.ini:1]: I haven't worked with github templates before, so I'm not sure if this needs to be backwards compatible or not.

File: Dockerfile [hunk 1]
```diff
@@ -36,10 +36,12 @@ RUN set -eux; \
 # https://getcomposer.org/doc/03-cli.md#composer-allow-superuser
 ENV COMPOSER_ALLOW_SUPERUSER=1
 
+ENV PHP_INI_SCAN_DIR="$PHP_INI_DIR/conf.d:$PHP_INI_DIR/app.conf.d"
```
Comment (bkosun) [inline Dockerfile:39]:
```suggestion
ENV PHP_INI_SCAN_DIR=":$PHP_INI_DIR/app.conf.d"
```

**Review by bkosun — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (bkosun) [reply Dockerfile:39]: In this case, PHP is configured with the `--with-config-file-scan-dir=/usr/local/etc/php/conf.d` option, so there is no need to specify this directory manually

Comment (leroy0211) [reply Dockerfile:39]: I tried that, but that didn't worked.  Seems to be working now. I'll apply it

File: frankenphp/conf.d/10-app.ini
Comment (dunglas) [reply frankenphp/conf.d/10-app.ini:1]: No need for backward compatibility!

Comment (leroy0211) [reply frankenphp/conf.d/10-app.ini:1]: Great!

**Comment by goulashsoup** [comment]
@dunglas I there a reason this is not merged yet?

**Review by dunglas — APPROVED** [review]
LGTM. WDYT @maxhelias

**Review by maxhelias — APPROVED** [review]
I agree with this 👍

**Comment by maxhelias** [comment]
Thanks @leroy0211 for fixing this and @bkosun for the review 😉

**Pull request merged**